### PR TITLE
Feat/new home

### PR DESCRIPTION
**Pull request by Y4nnikH**
Should be fine to merge this

### CODE REVIEW

File: src/pages/exercises.tsx [hunk 1]
```diff
@@ -232,57 +346,147 @@ const Exercises = (): JSX.Element => {
...
+                </span>
+              </span>
+            </div>
+          </Row>
           <Content
             style={{
               display: "flex",
               width: "100%",
-              padding: "10px",
+              padding: "00px",
```
Comment (Gamer92000) [inline src/pages/exercises.tsx:417]:
```suggestion
              padding: "0px",
```

File: src/pages/exercises.tsx [hunk 2]
```diff
@@ -232,57 +346,147 @@ const Exercises = (): JSX.Element => {
...
-              <Day
-                list={exercises}
-                name="sunday"
-                displayName={t(Translations.weekdays.sunday)}
-              />
-            </Row>
+              <Row
+                style={{
+                  padding: "10px 50px",
+                  // width: "100%",
```
Comment (Gamer92000) [inline src/pages/exercises.tsx:433]:
```suggestion
```

File: src/pages/exercises.tsx (hunk 1 above)
Comment (Gamer92000) [inline src/pages/exercises.tsx:417]:
```suggestion
              padding: "0px",
```

File: src/pages/exercises.tsx (hunk 2 above)
Comment (Gamer92000) [inline src/pages/exercises.tsx:433]:
```suggestion
```

File: src/styles/home.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  padding-right: initial;
+  flex-grow: 1;
+  width: initial;
+  margin-right: 10px;
+}
+
+.ant-progress-text {
+  text-align: right;
+  width: initial;
+}
```
Comment (Gamer92000) [inline src/styles/home.css:30]: Did you double check that this doesn't break other progress bars?

**Review by Gamer92000 — CHANGES_REQUESTED** [review]

File: src/styles/home.css (hunk 3 above)
Comment (Y4nnikH) [reply src/styles/home.css:30]: no

Comment (Y4nnikH) [reply src/styles/home.css:30]: The other ones don't seem to be affected by this 👍

**Review by Gamer92000 — APPROVED** [review]

**Pull request merged**